### PR TITLE
Update SCL_COMPAS.xsd file

### PR DESCRIPTION
**Pull request by gleizesDor**
- add attributes
- many attributes optional
- add documentation

### CODE REVIEW

File: scl-extension/src/main/resources/xsd/SCL_CoMPAS.xsd [hunk 1]
```diff
@@ -83,25 +81,6 @@ SPDX-License-Identifier: Apache-2.0
         </xs:restriction>
     </xs:simpleType>
 
-    <xs:simpleType name="tCompasLabel">
```
Comment (Sander3003) [inline scl-extension/src/main/resources/xsd/SCL_CoMPAS.xsd:86]: @pascalwilbrink do you know if we use this in the SCL dataservice?

**Comment by pascalwilbrink** [comment]
We make use of the CoMPAS:Label inside a Service, SCL-Data-Service. This cannot be removed

**Review by pascalwilbrink — CHANGES_REQUESTED** [review]
We make use of CoMPAS Labels inside a service, this cannot be removed

File: scl-extension/src/main/resources/xsd/SCL_CoMPAS.xsd (hunk 1 above)
Comment (gleizesDor) [reply scl-extension/src/main/resources/xsd/SCL_CoMPAS.xsd]: ok, I reverted this change

**Review by Sander3003 — APPROVED** [review]
Looks good to me

**Comment by gleizesDor** [comment]
> We make use of CoMPAS Labels inside a service, this cannot be removed
Please Could you see if the review is fine ? @pascalwilbrink ? The Pull request is blocking our developments since now two weeks.

**Review by pascalwilbrink — APPROVED** [review]

**Comment by pascalwilbrink** [comment]
> > We make use of CoMPAS Labels inside a service, this cannot be removed
> Please Could you see if the review is fine ? @pascalwilbrink ? The Pull request is blocking our developments since now two weeks.
> 

Approved it

**Pull request merged**